shared-helpers/src/locales/general.json

```diff
@@ -214,18 +214,18 @@
   "application.household.preferredUnit.subTitle": "Although unit sizes will typically be based on the number of people who will live in the unit, please provide your preferred unit size for determining your preference in this opportunity or establishing a waitlist (for this opportunity only).",
   "application.household.preferredUnit.title": "What unit sizes are you interested in?",
   "application.household.primaryApplicant": "Primary Applicant",
-  "application.name.dobHelper": "For example: 01 19 2000",
+  "application.name.dobHelper": "For example: January 19 2000",
```

Comment:
We'll have to use the old one here, this translation is assuming that the month selector is using the new design which does let you select month strings, but right now we just have a number input. We can keep these strings and their translations here and add another key that is the 01 version.